.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,35 @@
+name: auth-landing CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - name: Format code
+        run: npm run format
+      - name: Lint svelte files
+        run: npm run lint
+      - name: Generate production build
+        run: npm run export
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
GitHub access tokens are associated with an account, let's use a Deploy Key specific to this repo instead.